docs/monitoring/02-01-monitoring.md

```diff
@@ -2,32 +2,46 @@
 
-## The Prometheus Operator
+  For more details, [read](https://github.com/coreos/prometheus-operator) the official documentation.
 
-The Prometheus Operator is a CoreOS component integrated into Kyma that enables Prometheus deployments to be decoupled from the configuration of the entities they monitor. The task of the Operator is to ensure that Prometheus servers with the specified configuration are always running. If the developer does not specify a configuration for Prometheus instances, the Operator is able to generate and deploy one. The Prometheus instance is responsible for the monitoring of services.
+- **Prometheus** that collects metrics from Pods. The metrics are the time-stamped data that provide information on the running jobs, workload, CPU consumption, memory usage, and more. Pods can also contain applications with custom metrics, such as the total storage space available in the Minio server. Prometheus stores this polled data in a time-series database (TSDB) and runs rules over them to generate alerts if it detects any metric anomalies.
 
-## The Service Monitor
+  For more details, [read](https://prometheus.io/docs/introduction) the official documentation.
 
-The Service Monitor works in orchestration with the Prometheus resource that the Operator watches. It dictates to a Prometheus resource how to retrieve metrics and enables exposure of those metrics in a standard manner. It also specifies services the Prometheus instance should monitor. Using labels, the Prometheus resource includes a Service Monitor.  
+- **Grafana** that provides a dashboard and graph editor to visualize metrics collected from the Prometheus API. Grafana uses the query language called [PromQL](https://prometheus.io/docs/prometheus/latest/querying/basics/) to select and aggregate the metrics data from the Prometheus database. To access the Grafana UI, use the `https://grafana.{DOMAIN}` address, where {DOMAIN} is the domain of your Kyma cluster.
 
-## Monitored Data sources
+  For more details, [read](https://grafana.com/docs/guides/getting_started/) the official documentation.
 
-Prometheus contains the flexibility to poll data from a variety of sources. Virtual machines on which Kubernetes runs make time-stamped data available, reporting on jobs started, workload, CPU performance, capacity, and more. In this case, the Service Monitor watches the Kubernetes API master to detect any job creation. The job produces time-stamped data that Prometheus consumes.
+- **Alertmanager** that receives alerts from Prometheus and forwards this data to configured Slack or Victor Ops channels.
 
-Pods may contain applications with custom metrics that Prometheus can poll through the Prometheus exporter.
+  > **NOTE:** There are no notification channels configured in the default monitoring installation. The current configuration allows you to add either Slack or Victor Ops channels.
```

Comment:
Is it somewhere described how to add the channels? It could be useful to link to the instruction.

Reply:
We are going to create a tutorial for that. There is some info in the Alertmanager's README.md but I would not link to it yet since we are planning to rework that soon.